Jenkinsfile

```diff
@@ -10,20 +10,20 @@ elifePipeline {
         {
             stage 'Build images', {
                 checkout scm
-                dockerComposeBuild(commit)
+                sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml build"
             }
 
             stage 'Project tests', {
                 def coverallsToken = sh(script:'cat /etc/coveralls/tokens/profiles', returnStdout: true).trim()
                 withEnv(["COVERALLS_REPO_TOKEN=$coverallsToken"]) {
-                    dockerComposeProjectTests('profiles', commit, ['/srv/profiles/build/*.xml'])
+                    dockerComposeProjectTests 'profiles', commit
+                }
+                try {
+                    sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml up -d"
```

Comment:
this is possibly missing a wait for the migrations container to have run its course